docs/releases/2.7.3b2.rst

```diff
@@ -20,6 +20,8 @@ Details of changes
   usernames.
 - :djadmin:`refresh_scores` now recalculates user scores and accept
   multiple usernames.
+- :djadmin:`refresh_stats` option ``--calculate-checks`` was removed.  Use
```

Comment:
Actually several other options were dropped when removing old `refresh_stats` https://github.com/translate/pootle/commit/def9bf0029b4a35b73ad07e5cea094f61d6feb86#diff-083f311bbadfbeedbdbc2c70b180f198L46 But this change was already included in 2.7.1.